docs/docs/06-operations/10-docker-compose/15-three-servers.md

```diff
@@ -0,0 +1,123 @@
+      - LHS_SHOULD_CREATE_TOPICS=true
+      - LHS_HEALTH_SERVICE_PORT=1822
+      - LHS_INTERNAL_BIND_PORT=2011
+      - LHS_INTERNAL_ADVERTISED_HOST=lh-server-1
+      - LHS_INTERNAL_ADVERTISED_PORT=2011
+      - LHS_LISTENERS=PLAIN:2023
+      - LHS_ADVERTISED_LISTENERS=PLAIN://localhost:2023
+  littlehorse-2:
+    container_name: lh-server-2
+    image: ghcr.io/littlehorse-enterprises/littlehorse/lh-server:0.10.1
+    ports:
+      - "2024:2024"
+    environment:
+      - LHS_KAFKA_BOOTSTRAP_SERVERS=kafka:9092
+      - LHS_CLUSTER_ID=my-cluster
+      - LHS_INSTANCE_ID=2
+      - LHS_SHOULD_CREATE_TOPICS=true
+      - LHS_HEALTH_SERVICE_PORT=1822
+      - LHS_INTERNAL_BIND_PORT=2021
+      - LHS_INTERNAL_ADVERTISED_HOST=lh-server-2
+      - LHS_INTERNAL_ADVERTISED_PORT=2021
+      - LHS_LISTENERS=PLAIN:2024
+      - LHS_ADVERTISED_LISTENERS=PLAIN://localhost:2024
+  littlehorse-3:
+    container_name: lh-server-3
+    image: ghcr.io/littlehorse-enterprises/littlehorse/lh-server:0.10.1
+    ports:
+      - "2025:2025"
+    environment:
+      - LHS_KAFKA_BOOTSTRAP_SERVERS=kafka:9092
+      - LHS_CLUSTER_ID=my-cluster
+      - LHS_INSTANCE_ID=3
+      - LHS_SHOULD_CREATE_TOPICS=true
+      - LHS_HEALTH_SERVICE_PORT=1822
+      - LHS_INTERNAL_BIND_PORT=2031
+      - LHS_INTERNAL_ADVERTISED_HOST=lh-server-3
+      - LHS_INTERNAL_ADVERTISED_PORT=2031
+      - LHS_LISTENERS=PLAIN:2025
+      - LHS_ADVERTISED_LISTENERS=PLAIN://localhost:2025
+  dashboard:
+    container_name: lh-dashboard
+    image: ghcr.io/littlehorse-enterprises/littlehorse/lh-dashboard:master
+    environment:
+      LHC_API_HOST: lh-server-1
+      LHC_API_PORT: 2023
+      LHD_OAUTH_ENABLED: false
+    restart: on-failure
+    healthcheck:
+      test: curl -f localhost:3000
+      interval: 5s
+    ports:
+      - "8080:3000"
+```
+
+## Using the Example
+
+Once you have created the LH Cluster using Docker Compose, the next step is to access it. For example, you can use one of our [quickstarts](../../05-developer-guide/00-install.md#get-started) with the following configurations.
+
+The dashboard can be accessed at [`http://localhost:8080`](http://localhost:8080).
+
+You can connect to any of the three LittleHorse server instances in the cluster. In this example, the dashboard is configured to communicate with the server on port `2023`, but you could also connect with the servers on ports `2024` or `2025`.
+
+### Worker Configs
+
+Workers are configured when you create an `LHConfig` object according to [our documentation](../../05-developer-guide/02-client-configuration.md#creating-the-lhconfig). You need to pass in certain properties (either directly to the `LHConfig` constructor, through a `Properties` file, or through environment variables).
+
+For this example, your workers should be configured as follows:
+
+```
+LHC_API_HOST=localhost
+
+// You can address any of the three LittleHorse server ports
+LHC_API_PORT=2023
+
+LHW_SERVER_CONNECT_LISTENER=PLAIN
+```
+
```

Comment:
```suggestion

Even though your worker config only specifies one port to connect to via `LHC_API_PORT`, your workers will actually communicate with the entire cluster. This is because the Task Worker uses a discovery protocol in which it asks the bootstrap host (specified by `LHC_API_HOST` and `LHC_API_PORT`) for which servers to connect to in order to execute tasks.

:::tip
In production, it's recommended for your `LHC_API_HOST` and `LHC_API_PORT` to point to a load balancer that balances requests across all instances.
:::

```